src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -296,54 +296,34 @@ public void ProcessStart_UseShellExecute_OnUnix_ValidVerbs(string verb, bool isV
             }, verb ?? "<null>", isValid.ToString(), options).Dispose();
         }
 
-        [Theory, InlineData("vi")]
-        [PlatformSpecific(TestPlatforms.Linux)]
-        [OuterLoop("Opens program")]
-        public void ProcessStart_OpenFileOnLinux_UsesSpecifiedProgram(string programToOpenWith)
+        [Fact]
+        public void ProcessStart_OnLinux_UsesSpecifiedProgram()
         {
-            if (IsProgramInstalled(programToOpenWith))
-            {
-                string fileToOpen = GetTestFilePath() + ".txt";
-                File.WriteAllText(fileToOpen, $"{nameof(ProcessStart_OpenFileOnLinux_UsesSpecifiedProgram)}");
-                using (var px = Process.Start(programToOpenWith, fileToOpen))
-                {
-                    Assert.Equal(programToOpenWith, px.ProcessName);
-                    px.Kill();
-                    px.WaitForExit();
-                    Assert.True(px.HasExited);
-                }
-            }
-            else
+            string program = "sleep";
+
+            using (var px = Process.Start("sleep", "30"))
```

Comment:
Nit: `"sleep"` => `Program`

Reply:
Wondering if we might want to make 30 be longer, since effectively we want the process to run forever until killed.  Is 30 what we use elsewhere in the process tests for such things?

Reply:
I've updated to 60 sec to match default value of RemoteExecutor.FailWaitTimeoutMilliseconds.